core/src/main/scala/fr/davit/akka/http/metrics/core/MeterStage.scala

```diff
@@ -60,7 +67,10 @@ private[metrics] class MeterStage(metricsHandler: HttpMetricsHandler)
 
       override def onPush(): Unit = {
         val request = grab(requestIn)
-        val id      = request.getAttribute(HttpMetrics.TraceId).get
+        val id = request.attribute(HttpMetrics.TraceId) match {
+          case Some(id) => id
+          case _        => throw MissingTraceIdException
```

Comment:
When possible we should avoid throwing. In this case we can use the `failStage` method
```suggestion
          case Some(id) => 
            pending += id -> request
            push(requestOut, metricsHandler.onRequest(request))
          case _ => 
            failStage(MissingTraceIdException)
```

Reply:
I am all hands for not throwing exception. Unfortunately, I am not so strong in Akka streams. Will change it